packages/fxa-auth-server/lib/senders/emails/templates/postSigninRecoveryCodes/index.mjml

```diff
@@ -0,0 +1,27 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="postSigninRecoveryCodes-title">Your backup authentication code was used to sign in</span>
```

Comment:
Since this title seems to be the only difference between these, I almost want to suggest moving the entire rest of the email to be in its own partial, but eh. Probably not worth it in case these change later.

Reply:
I wondered too, but considering this is a different flow and we might want to make changes later I'd prefer to keep these separate.